src/plugins/elasticsearch/index.js

```diff
@@ -23,7 +23,8 @@ module.exports = function (kibana) {
           cert: Joi.string(),
           key: Joi.string()
         }).default(),
-        minimumVerison: Joi.string().default('1.4.4')
+        apiVersion: Joi.string().default('master'),
```

Comment:
I think this should be changed to 2.0, but support for the 2.0 api version has not been released yet.

It is on my short todo list, and I expect to get to it today.